src/foraging_gui/Foraging.py

```diff
@@ -869,7 +890,8 @@ def _GetSettings(self):
         window_title = '{}'.format(self.current_box)
         self.window_title = window_title
 
-
+        # Get the commit hash of the current version of this Python file
+        self.commit_ID, self.current_branch, self.repo_url=log_git_hash()
```

Comment:
This results in `log_git_hash` getting called twice, once at start up, and once here. We should avoid calling the exact function twice

Reply:
It's more elegant to call it only once. I updated the code